csrc/python_frontend/python_bindings.cpp

```diff
@@ -733,6 +733,16 @@ void initNvFuserPythonBindings(PyObject* module) {
                   " was neither symbolic(-1), zero_element(0), broadcast(1), or static(>1).");
             }
 
+            if (contiguity.empty()) {
+              for (const auto dim_size : shape) {
+                if (dim_size == 1) {
+                  contiguity.emplace_back(std::nullopt);
+                } else {
+                  contiguity.emplace_back(true);
```

Comment:
nitpick: should we default to `false` though....

Reply:
Good point -- that'll be consistent with https://github.com/NVIDIA/Fuser/blob/5cb0bb4a37e1950cf1a4dda809aa0b1744bc178a/csrc/ir/nodes.cpp#L3267.